CI/README.md

```diff
@@ -0,0 +1,4 @@
+## Usage for a CI run of cerberus tests: 
```

Comment:
Maybe add the list of tests supported and how the gold, time delta percentage between the master and commit branches is calculated to the readme?

Reply:
yes will do